tests/validation/validate-form.test.js

```diff
@@ -66,3 +66,58 @@ test("it validates all the form fields including the ones in sub forms", (t) =>
   });
   t.true(isValid, "The source data contains valid listing item data");
 });
+
+test("it supports validating specific severity levels", (t) => {
```

Comment:
might be nice to add a testcase that explicitly checks that validations without a severity level are still interpreted as error, i believe that the way the test form is set up, you can't have this case though :/

Reply:
You mean like test a case where there is _only_ a validator without the `sh:severity` predicate? I think that might already be implicitly covered by the first test.

Yea, I'm pretty new to testing / organization of fixtures etc. Maybe I shouldn't try to reuse fixtures, even though it causes some "duplication"?

Reply:
I always end up with the same problem for this kind of end-to-end tests but i still like them more than when i'm mocking everything and just unit testing things, so up till now i feel it is worth the effort. Having separate fixtures feels like duplication but keeps your tests independent which is probably nice, though I can't say that I haven't also reused fixtures before :see_no_evil: 

you are completely correct though, i had missed the first testcase, which does cover the case of no severity being specified of course!